openwisp_firmware_upgrader/admin.py

```diff
@@ -347,29 +346,11 @@ class Media:
         js = ['admin/js/jquery.init.js', 'firmware-upgrader/js/device-firmware.js']
         css = {'all': ['firmware-upgrader/css/device-firmware.css']}
 
-    def _get_image_queryset(self, device):
-        # restrict firmware images to organization of the current device
-        qs = (
-            FirmwareImage.objects.filter(
-                build__category__organization_id=device.organization_id
-            )
-            .order_by('-created')
-            .select_related('build', 'build__category')
-        )
-        # if device model is defined
-        # restrict the images to the ones compatible with it
-        if device.model and device.model in REVERSE_FIRMWARE_IMAGE_MAP:
-            qs = qs.filter(type=REVERSE_FIRMWARE_IMAGE_MAP[device.model])
-        # if DeviceFirmware instance already exists
-        # restrict images to the ones of the same category
-        if not self.instance._state.adding:
-            self.instance.refresh_from_db()
```

Comment:
@nemesifier I don't understand why this was present here. Do you remember?

Reply:
Unfortunately I don't remember..